automatic/index.html

```diff
@@ -6,7 +6,7 @@
     <form action="charge.php" method="POST">
       <script
         src="https://checkout.razorpay.com/v1/checkout.js"
-        data-key="rzp_test_WyK93y9mvps7SN"
+        data-key="<Your Key Id>"
```

Comment:
1. Move this to `config.php` as discussed
2. Maybe use `index.php?checkout=automatic|manual`